rmw_connextdds_common/src/common/rmw_context.cpp

```diff
@@ -991,7 +1013,7 @@ rmw_api_connextdds_init(
   ctx->optimize_large_data = '\0' == disable_optimize_large_data_env[0];
 #endif /* RMW_CONNEXT_DEFAULT_LARGE_DATA_OPTIMIZATIONS */
 
-  /* Lookup and configure initial peer from environment */
+    /* Lookup and configure initial peer from environment */
```

Comment:
?

Reply:
After further review, this variable should be deprecated now that `ROS_STATIC_PEERS` is a thing, since they serve the same purpose.

We could add a note to the description in the README, but it can also be addressed by a separate PR.

Reply:
If @wjwwood was commenting on the unnecessary addition of the space, I think that was probably a fluke from my IDE's auto-formatter.

But I agree that the `RMW_CONNEXT_ENV_INITIAL_PEERS` env var could be deprecated now since it's redundant with the `ROS_STATIC_PEERS` env var.

Reply:
That's right, I was commenting on the whitespace, sorry I should have been more specific.

As for the idea of deprecating the other env var, I'm fine with that too, but should we do that separately? Or would you prefer to do that in this pr @asorbini? I really think it's up to you.

Reply:
I think it's better to address that deprecation in a separate PR.